CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 
 - [#636](https://github.com/influxdata/kapacitor/pull/636): Change HTTP logs to be in Common Log format.
 - [#652](https://github.com/influxdata/kapacitor/pull/652): Add optional replay ID to the task API so that you can get information about a task inside a running replay.
+- [#680](https://github.com/influxdata/kapacitor/pull/680): Add Telegram Alerting option
```

Comment:
Also, can you rebase off latest master and move this line to the beta3 section since beta2 has now been released?